config/pmd/pmd.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    name="PMD rules"
+    xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+    xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+
+    <description>Ruleset Android application</description>
+
+    <exclude-pattern>.*/R.java</exclude-pattern>
+    <exclude-pattern>.*/gen/.*</exclude-pattern>
+
+    <rule ref="rulesets/java/unnecessary.xml"/>
```

Comment:
Do these rules map to any of our documentation anywhere? Either the existing Checkstyle config or the new standards coming out on Confluence?

Reply:
I haven't documented for these standards yet but I will do it along with creating a Validation Gradle plugin encapsulating PMD, Findbugs and Checkstyle